std/encoding.d

```diff
@@ -619,27 +628,7 @@ struct CodeUnits(E)
         s = encode!(E)(d);
     }
 
-    int opApply(scope int delegate(ref E) dg)
-    {
-        int result = 0;
-        foreach(E c;s)
-        {
-            result = dg(c);
-            if (result != 0) break;
-        }
-        return result;
-    }
-
-    int opApplyReverse(scope int delegate(ref E) dg)
-    {
-        int result = 0;
-        foreach_reverse(E c;s)
-        {
-            result = dg(c);
-            if (result != 0) break;
-        }
-        return result;
-    }
+    alias s this;
```

Comment:
CodeUnits is part of the public interface, albeit undocumented, so I can't eliminate it in one release. I've deprecated the struct and the one function that creates it and reverted my other changes. Internally I switched std.encoding to use encode(dchar), which CodeUnits calls internally.

However, I don't know how long something should be deprecated before removal. A year? 18 months?